connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -103,6 +104,12 @@ class PlanGenerationTestSuite
     val client = SparkConnectClient(InProcessChannelBuilder.forName("/dev/null").build())
     session =
       new SparkSession(client, cleaner = SparkSession.cleaner, planIdGenerator = new AtomicLong)
+    if (regenerateGoldenFiles) {
+      if (queryFilePath.toFile.exists()) {
```

Comment:
nit: maybe we can use `Utils.deleteRecursively`

Reply:
Done.